front/src/layouts/layout-footer/layout-footer.layout.css

```diff
@@ -0,0 +1,60 @@
+div.main,
+div.footer-color-background {  
+  display: flex;
+  flex-direction: row;                        
+  align-items: center;
+  justify-content: space-between; 
+}
+
+div.main {
+  height: 125px;
+}
+
+div.logo {
+  margin: 5px 0px 10px 80px;
+  padding: 5px 0px 10px;   
+}
+
+div.stores {
+  margin-left: 80px;   
+}
+
+div.footer-color-background {
+  height: 57px; 
+}
+
+span {
```

Comment:
either it is scoped styles it is really bad way to manage margin and font via changing style for span